modules/etl/camunda/element-templates/liferay-webhook-connector-boundary.json

```diff
@@ -0,0 +1,299 @@
+			"label": "Authorization Type",
+			"type": "Dropdown",
+			"value": "NONE"
+		},
+		{
+			"binding": {
+				"name": "inbound.auth.apiKey",
+				"type": "zeebe:property"
+			},
+			"condition": {
+				"equals": "APIKEY",
+				"property": "inbound.auth.type",
+				"type": "simple"
+			},
+			"description": "Expected secret.",
+			"feel": "optional",
+			"group": "authorization",
+			"id": "inbound.auth.apiKey",
+			"label": "Secret",
+			"optional": false,
+			"type": "String"
+		},
+		{
+			"binding": {
+				"name": "inbound.auth.apiKeyLocator",
+				"type": "zeebe:property"
+			},
+			"condition": {
+				"equals": "APIKEY",
+				"property": "inbound.auth.type",
+				"type": "simple"
+			},
+			"constraints": {
+				"notEmpty": true
+			},
+			"description": "A FEEL expression that extracts API key from the request. <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/liferay/#how-to-configure-secret'>See documentation</a>.",
+			"feel": "required",
+			"group": "authorization",
+			"id": "inbound.auth.apiKeyLocator",
+			"label": "Secret Locator",
+			"optional": false,
+			"type": "String",
+			"value": "=get value(request.headers, \"x-api-key\")"
+		},
+		{
+			"binding": {
+				"name": "inbound.responseExpression",
+				"type": "zeebe:property"
+			},
+			"description": "Expression used to generate the HTTP response.",
+			"feel": "required",
+			"group": "webhookResponse",
+			"id": "inbound.responseExpression",
+			"label": "Response Expression",
+			"optional": true,
+			"type": "Text"
+		},
+		{
+			"binding": {
+				"name": "activationCondition",
+				"type": "zeebe:property"
+			},
+			"description": "Condition under which the Connector triggers. Leave empty to catch all events.",
+			"feel": "required",
+			"group": "activation",
+			"id": "activationCondition",
+			"label": "Activation Condition",
+			"optional": true,
+			"type": "String"
+		},
+		{
+			"binding": {
+				"name": "consumeUnmatchedEvents",
+				"type": "zeebe:property"
+			},
+			"group": "activation",
+			"id": "consumeUnmatchedEvents",
+			"label": "Consume Unmatched Events",
+			"tooltip": "Unmatched events are rejected by default, allowing the upstream service to handle the error. Check this box to consume unmatched events and return a success response",
+			"type": "Boolean",
+			"value": true
+		},
+		{
+			"binding": {
+				"name": "correlationKey",
+				"type": "bpmn:Message#zeebe:subscription#property"
+			},
+			"constraints": {
+				"notEmpty": true
+			},
+			"description": "Sets up the correlation key from process variables.",
+			"feel": "required",
+			"group": "correlation",
+			"id": "correlationKeyProcess",
+			"label": "Correlation Key (Process)",
+			"type": "String"
+		},
+		{
+			"binding": {
+				"name": "correlationKeyExpression",
+				"type": "zeebe:property"
+			},
+			"constraints": {
+				"notEmpty": true
+			},
+			"description": "Extracts the correlation key from the incoming message payload.",
```

Comment:
It's hard to say what this should be without understanding it, but it's not a complete sentence.

Reply:
This sentence  is literally copied from other Camunda connectors